lib/mix/lib/mix/tasks/compile.protocols.ex

```diff
@@ -98,11 +97,11 @@ defmodule Mix.Tasks.Compile.Protocols do
   end
 
   defp consolidation_paths do
-    filter_otp(:code.get_path, :code.lib_dir)
+    filter_otp(:code.get_path(), :code.lib_dir())
   end
 
   defp filter_otp(paths, otp) do
-    Enum.filter(paths, &(not :lists.prefix(&1, otp)))
+    Enum.filter(paths, &not :lists.prefix(&1, otp))
```

Comment:
I'm not sure about this line, there is no need braces?

Reply:
Nope, parens are not needed. I would still prefer `&not(...)` but semantically and syntactically this is correct.

Reply:
I also think that `Enum.filter(paths, &not(:lists.prefix(&1, otp)))` would be better, but formatter does not agree :)